### DISCUSSION TITLE
Birdnet pi vs PUC

### DISCUSSION THREAD

**Discussion by waynecarnold** [mcguirepr89/BirdNET-Pi#1207] [General]
I have the new puc ( from birdnet.com) and use it when hiking. Today i placed it beside my pi as i wanted to see if it also detected a spotted crake which is not known to be in my area however the pi identified it.
Over a period of 15 minutes the puc identified pine siskin, ring billed gull, blue jay however the pi identified goldfinch blue jay and mallard. 

Which to believe? Or both are right? I figure the quality of mic may lead to differences but thought it means some birds would be missed on one system vs other. Was not expecting totally different identifications. After some thought i am figuring it is probably due to some of the pi customizations in the birdnet app like confidence settings etc. 

Just putting this out there to see what others may think

**Comment by waynecarnold** [comment]
Another thought, since the pi analyzes locally and removes tracks with human voice, wheras the puc uploads everything and analyzes on the server, this may result in some missed detections from the pi. I guess the puc can take advantage of more recent updates whereas the pi is not being updated any longer.

**Reply by MueJosh** [reply]
> I guess the puc can take advantage of more recent updates whereas the pi is not being updated any longer. 

Kinda wrong :) 
[Nachtzuster BirdNET-Pi](https://github.com/Nachtzuster/BirdNET-Pi)

**Reply by alexbelgium** [reply]
Indeed ; the pi is quite active on this fork and the owner has made many optimisations.

However, other criteria could affect : threshold of detection (if active), sensitivity... and indeed mic quality as PUC is optimized while for the pi there is a significant impact on the parts used to made the system.

And in the end the key element being - which one was telling the truth, was it false negative from one system, or false positive from the other ?

**Comment by stevedee** [comment]
My suggestion would be to use a 3rd system for arbitration (e.g. it may support the detections on one system over the other).
There are a number of apps for mobile phones including BirdNet & Merlin. But my favourite is whoBIRD which saves all detections, allowing you to download a CSV file and do your own filtering (e.g. in a spreadsheet with your chosen _confidence_ %):-
 https://github.com/woheller69/whoBIRD
